frontend/tailwind.config.js

```diff
@@ -7,10 +7,10 @@ export default {
         "cta-blue": "#002795",
         "cta-blue-hover": "#1138a6",
 
-        "main-green": "#97a65D",
+        "main-green": "rgba(132, 150, 77, 0.8)",
```

Comment:
This could cause issues since it's a bit transparent. It will shift colors depending on what's behind it. Could we find the same shade without it being transparent?